packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -204,6 +211,12 @@ class UnreferencedStateTracker {
 /**
  * The garbage collector for the container runtime. It consolidates the garbage collection functionality and maintains
  * its state across summaries.
+ *
+ * Node - represented as nodeId, it's a node on the GC graph
+ * Reference - a route with a parent, think `nodeId` & `Route`
+ * Route - a node that is reachable from another node, think `nodeId`
+ * Outbound - a route or reference going from parent to child
```

Comment:
There is no parent child relationship in a graph. Its just a set of nodes and dependencies.

Reply:
So, only node, outbound route and graph are the only terms we need. You can correlate them to vertices and directed edges in a directed graph. Maybe even write an example here and show what the different terms look like w.r.t. graph

Reply:
A comment with a diagram like this - https://github.com/microsoft/FluidFramework/blob/b84ea277bad1d7133df989c8e5cd64973b3d6bc9/packages/runtime/container-runtime/src/test/dataStoreCreation.spec.ts#L29